frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditLayoutSetName/EditLayoutSetName.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { StudioToggleableTextfield } from '@studio/components';
+import { KeyVerticalIcon } from '@studio/icons';
+import { useBpmnApiContext } from '../../../../contexts/BpmnApiContext';
+import { Paragraph } from '@digdir/designsystemet-react';
+import { useValidateLayoutSetName } from 'app-shared/hooks/useValidateLayoutSetName';
+import { useBpmnContext } from '../../../../contexts/BpmnContext';
+
+interface EditLayoutSetNameProps {
+  existingLayoutSetName: string;
+}
+export const EditLayoutSetName = ({
+  existingLayoutSetName,
+}: EditLayoutSetNameProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const { bpmnDetails } = useBpmnContext();
+  const { layoutSets, mutateLayoutSetId } = useBpmnApiContext();
+  const { validateLayoutSetName } = useValidateLayoutSetName();
+
+  const handleOnLayoutSetNameBlur = (event: React.ChangeEvent<HTMLInputElement>): void => {
+    const newName = event.target.value;
+    if (newName === existingLayoutSetName) return;
```

Comment:
I noticed this line is not covered by tests, could you add a test case to cover this?